### ISSUE TITLE
Problems with audio Volume control

### ISSUE THREAD

**Issue by darkoverlordofdata** [helloSystem/Utilities#164]
I’ve run into problems with the audio

1) When I try to use YouTube (I’m using Chromium, Falkon is blocked when I try to log in to my Google account), the volume is extremely low. I have to turn it up to 100% and also turn up the Volume control on the menu panel to 100%, and then I can just barely hear youtube.

2) There is only Play, no Rec level slider. I used to see a Rec slider on the Volume control in  previous versions of helloSystem. I ran into this trying to record live music - I’ve started a discussion for that : [Using helloSystem to create music](https://github.com/helloSystem/hello/discussions/502)

**Comment by darkoverlordofdata** [comment]
I don’t think that will help; I actually found this issue also happening with the DeadBeef CD player, so I think it's the os audio settings, not the specific application. On the FreeBSD forums, I found advice to set hw.snd.vpc_0db to 1. Currently, helloSystem has this set to 20, and the FreeBSD default setting is 45.  I set this value to 1 as suggested, and the issue has cleared up.

To set this persistently, edit /etc/sysctl.conf, and add ‘hw.snd.vpc_0db=1’ in the file.

This issue is resolved for me.

**Comment by probonopd** [comment]
See

* https://github.com/helloSystem/ISO/issues/8

Doesn't the sound become crappy when you set it to 1 and then move the volume slider to the max?
